exporter/opentelemetry-exporter-prometheus/src/opentelemetry/exporter/prometheus/__init__.py

```diff
@@ -157,7 +158,10 @@ def _translate_to_prometheus(
         label_keys = []
         for key, value in metric.attributes.items():
             label_keys.append(self._sanitize(key))
-            label_values.append(str(value))
+            if isinstance(value, (list, tuple)):
+                label_values.append(dumps(value, default=str))
```

Comment:
Curious what does this `default=str` do?

Reply:
It is a [function that gets called for objects that can’t otherwise be serialized](https://docs.python.org/3/library/json.html#json.dump).